### DISCUSSION TITLE
Maximum file size

### DISCUSSION THREAD

**Discussion by kukuhprastyono** [DefectDojo/django-DefectDojo#10514] [Q&A] [answered]
i use /api/v2/reimport-scan to send Trivy Scan result of Go image, but at the first time i always got upstream request timeout error
is error caused by file size to big? what is maximum file size that can be sent?

**Comment by manuel-sommer** [comment, marked as answer]
Hi @kukuhprastyono,

there is a setting in settings.dist.py to adjust the maximum file size which can be uploaded. See here: https://github.com/DefectDojo/django-DefectDojo/blob/659f50b129cef6568028c0422fe6157539012a74/dojo/settings/settings.dist.py#L281

In order to set the env settings, please use a local_settings.py file like it is written https://documentation.defectdojo.com/getting_started/configuration/ and discribed also here: https://github.com/DefectDojo/django-DefectDojo/blob/659f50b129cef6568028c0422fe6157539012a74/dojo/settings/settings.dist.py#L2 
Otherwise, you will get problems with the sha sum of https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/settings/.settings.dist.py.sha256sum

Does this help you?

Best,

Manuel

**Reply by kukuhprastyono** [reply]
Thanks @manuel-sommer fo your answer
but is any issue if i set limit to big?

**Reply by manuel-sommer** [reply]
It could have other side effects, e.g. your nginx times out if the size is too big to handle within a specific time. Then, you have to adjust other parameters (e.g. the nginx time out). Depends also on your deployment, there is no general answer for it. Just try it out.

**Reply by kukuhprastyono** [reply]
Okay
